kernel/kernel-util/src/main/java/org/sakaiproject/util/ResourceLoader.java

```diff
@@ -257,7 +257,16 @@ public String getLocaleDisplayName(Locale loc) {
 
 		return displayName.toString();
 	}
-   
+
+	public String getOrientation(Locale loc) {
+		String [] locales = ServerConfigurationService.getStrings("locales.rtl");
+		if (locales==null) {
+			locales = new String[]{"ar","dv","fa","ha","he","iw","ji","ps","ur","yi"};
+		}
+		HashSet<String> rtl = new HashSet<String>(Arrays.asList(locales));
```

Comment:
```org.sakaiproject.util.ArrayUtil#contains(Object[], Object)``` is a better way of doing this as putting the array into a HashSet means you are iterating on it anyway.

With such a small number of items the performance of scanning the array should be very good anyway.